WooCommerce/Classes/ViewRelated/Reviews/ReviewReplyViewModel.swift

```diff
@@ -27,8 +52,33 @@ final class ReviewReplyViewModel: ObservableObject {
     /// Use this method to send the reply and invoke a completion block when finished
     ///
     func sendReply(onCompletion: @escaping (Bool) -> Void) {
-        // TODO: Call CommentAction.replyToComment to send the reply to remote
-        // Set `performingNetworkRequest` to true while the request is being performed
+        guard newReply.isNotEmpty else {
+            return
+        }
+
+        let action = CommentAction.replyToComment(siteID: siteID, commentID: reviewID, content: newReply) { [weak self] result in
+            guard let self = self else { return }
+
+            self.performingNetworkRequest.send(false)
+
+            switch result {
+            case .success(let status):
+                // If the comment isn't approved, log it (to help support)
+                if status != .approved {
+                    DDLogInfo("Reply to product review succeeded with comment status: \(status)")
+                }
+
+                self.displayReplySuccessNotice()
```

Comment:
Small nit, I think that this probably should be fired from the invoking `ViewController` as it's purely UI logic!